gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/ExecutionHistoryService.groovy

```diff
@@ -47,7 +47,15 @@ class ExecutionHistoryService {
     Preconditions.checkNotNull(app)
     RequestContext requestContext = RequestContext.get()
     def command = HystrixFactory.newListCommand("pipelineExecutionHistory", "getPipelinesForApp-$app") {
-      orcaServiceSelector.withContext(requestContext).getPipelines(app, limit, statuses, expand)
+      try { 
+     orcaServiceSelector.withContext(requestContext).getPipelines(app, limit, statuses, expand)
+      }catch(Exception e){
+			
+                e.message
```

Comment:
Hystrix does useful things (retries, circuit breaking) with Retrofit exceptions, wouldn't want to intercept one like this. This also isn't relevant to Orca's behavior when the `Queue` is available but `ExecutionRepository` isn't.

Please also make indents and other whitespace follows our standards when opening PR's.